std/format.d

```diff
@@ -2621,6 +2624,8 @@ if (is(Unqual!T : creal) && !is(T == enum) && !hasToString!(T, Char))
     }
 }
 
+version(TestComplex)
```

Comment:
https://issues.dlang.org/show_bug.cgi?id=18250 - I didn't know a better way to workaround it.